test/functional/tm/gtm/test_rule.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2015 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+
+from f5.bigip.resource import MissingRequiredCreationParameter
+from f5.bigip.tm.gtm.rule import Rule
+from requests.exceptions import HTTPError
+
+
+RULE = '''when LB_SELECTED {
+   set wipHost [LB::server addr]
+}
+'''
+
+
+def delete_rule(bigip, name, partition):
```

Comment:
Could you also do me another small favor and create another fixture alongside the 'bigip', which is called 'management_root' or 'mgmt_root' which has the new path, and use that instead of the old root?

The new fixture would be in here: https://github.com/F5Networks/f5-common-python/blob/development/conftest.py